DigitalLearningSolutions.Web/ViewModels/Login/LoginViewModel.cs

```diff
@@ -9,6 +9,14 @@ public LoginViewModel()
             Username = string.Empty;
             Password = string.Empty;
             RememberMe = false;
+            ReturnUrl = string.Empty;
+        }
+        public LoginViewModel(string? returnUrl)
```

Comment:
Super NIP, but missing an empty line above this (formatter should in theory catch this..).
Also, is there a reason we need to set the ReturnUrl in the constructor above given that it's nullable?